js/combat.js

```diff
@@ -0,0 +1,111 @@
+function updateHealth(menu){
```

Comment:
regrouped the combat stuff into a separate file to keep index.js more about state management and higher functions.